xc_fasm/fasm2frames.py

```diff
@@ -271,7 +297,8 @@ def check_for_pudc_b(set_feature):
     if f_out is not None:
         dump_frm(f_out, frames)
 
-    return frames
+    return (frames, clb_start_address, clb_end_address, bram_start_address,
```

Comment:
Can you please make this a namedtuple?  This is a lot of return arguments, and a namedtuple I think handles this better.